assets/js/util/i18n.js

```diff
@@ -57,6 +57,36 @@ export const prepareSecondsForDisplay = ( seconds ) => {
 	return returnString.trim();
 };
 
+/**
+ * Converts seconds into an array with the following values [HH, MM, SS, MS] as described below.
+ * HH: hours.
+ * MM: minutes.
+ * SS: seconds.
+ * MS: milliseconds.
+ *
+ * For example, passing 196.385 returns [ 0, 3, 16, 385 ].
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} seconds The number of seconds.
+ * @return {Array} Array containing the hours, minutes, seconds and milliseconds.
+ *
+ */
+export const prepareSecondsArray = ( seconds ) => {
+	seconds = parseInt( seconds, 10 );
+
+	if ( isNaN( seconds ) || 0 === seconds ) {
+		return [ 0, 0, 0, 0 ];
+	}
+
+	return [
+		Math.floor( seconds / 60 / 60 ),
+		Math.floor( ( seconds / 60 ) % 60 ),
+		Math.floor( seconds % 60 ),
+		parseInt( parseFloat( seconds ).toFixed( 3 ).slice( -3 ) ),
+	];
+};
+
```

Comment:
The logic here seems fine, but two notes:

1. I don't think this belongs in the `i18n` utils file. It's not i18n-related—it can either go in a date/time-related file or in its own file, whichever makes more sense.
2. It'd be good to add a few simple unit tests for this to ensure it's behaving as expected

Reply:
@tofumatt I've created a new file, `convert-time.js` and moved the function over there. I also added the tests.